### ISSUE TITLE
The requested module '@stream-io/node-sdk' does not provide an export named 'StreamClient'

### ISSUE THREAD

**Issue by isaac-merkos** [GetStream/stream-node#60]
I am importing the client like this
```ts
import { StreamClient } from "@stream-io/node-sdk";
```

And running my application like this
```bash
node --enable-source-maps --no-warnings --loader ts-node/esm src/index.ts
```

And I am getting this error which causes my application to crash
```text
import { StreamClient } from "@stream-io/node-sdk";
         ^

SyntaxError: The requested module '@stream-io/node-sdk' does not provide an export named 'StreamClient'
    at ModuleJob._instantiate (node:internal/modules/esm/module_job:132:21)
    at async ModuleJob.run (node:internal/modules/esm/module_job:214:5)
    at async ModuleLoader.import (node:internal/modules/esm/loader:329:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:113:12)
```

relevant output from `npm list`
```text
@stream-io/node-sdk@0.4.3
ts-node@10.9.2
```
node v20.10.0

There are no typescript errors and no running it with bun seems to work but I can't use bun for other reasons

Here is my `tsconfig.json`
```json
{
  "extends": "./tsconfig-tsnode20.json",
  "compilerOptions": {
    "checkJs": true,
    "lib": ["ES2022"],
    "target": "ES2022",
    "sourceMap": true,
    "isolatedModules": true,
    "esModuleInterop": true,
    "resolveJsonModule": true,
    "outDir": "build"
  },
  "ts-node": {
    "transpileOnly": true
  },
  "include": ["./src/**/*", "./tools/**/*", "./types/**/*"]
}
```

**Comment by szuperaz** [comment]
Sorry for the late reply, we're investigating it and get back to you.

**Comment by szuperaz** [comment]
The issue has been fixed in `@stream-io/node-sdk@0.4.4`.

I will close this ticket, but feel free to reopen it if the fix doesn't work.

**Comment by isaac-merkos** [comment]
@szuperaz Can confirm it now works, thanks
